iron-validator-behavior.html

```diff
@@ -17,36 +17,13 @@
    * Use `Polymer.IronValidatorBehavior` to implement a custom input/form validator. Element
    * instances implementing this behavior will be registered for use in elements that implement
    * `Polymer.IronValidatableBehavior`.
-   * 
+   *
    * @demo demo/index.html
    * @polymerBehavior
    */
   Polymer.IronValidatorBehavior = {
-
-    properties: {
-
-      /**
-       * Namespace for this validator.
-       */
-      validatorType: {
```

Comment:
They were unused. `validatorType` was always set to `validator`, and changes to it were not observed, so it was useless as a property (also, it was only used in `ready`, so if you ever did change it, it was too late)